doc/basic-first-aid.rst

```diff
@@ -0,0 +1,150 @@
+###############
+Basic First Aid
+###############
+
+These suggestions, while more of a shotgun approach, tend to fix the majority of reported issues just on their own.
+
+
+
+=================================
+Recommended troubleshooting steps
+=================================
+
+#. **Restart Spyder**, and try what you were doing before again.
+
+#. **Upgrade Spyder** to the latest release, and you might find your issue is resolved (along with new features, enhancements, and other bug fixes).
+   Minor releases come out every couple months, so unless you've updated recently, there is a good chance your version isn't the latest.
+   You can find out with the :guilabel:`Check for updates` command under the :guilabel:`Help` menu.
+
+   .. image:: images/basic-first-aid/basic-first-aid-updates.png
+      :alt: Spyder showing view internal console option
+
+   To perform the update with Conda (strongly recommended), from your terminal (or Anaconda Prompt on Windows) run:
+
+   .. code-block:: bash
+
+      conda update anaconda
+      conda update spyder
+
+#. **Update Spyder's dependencies and environment**, either by installing the latest version of your distribution (e.g. the recommended Anaconda), or with the relevant "update all" command in your terminal (or Anaconda Prompt on Windows).
+   To get the latest stable version of everything using Conda, you can run:
+
+   .. code-block:: bash
+
+      conda update qt pyqt spyder-kernels ipython ipykernel jupyter_client jupyter_core pyzmq
+
+#. **Restart your machine**, in case the problem lies with a lingering process or another such issue.
+
+#. **Restore Spyder's config files** to their defaults, which solves a huge variety of Spyder issues.
+   From your terminal (or Anaconda Prompt on Windows), run:
+
+   .. code-block:: bash
+
+      spyder --reset
+
+   .. note::
+
+      This will reset your preferences, as well as any custom keyboard shortcuts or syntax highlighting schemes.
+      If you particularly care about any of those, you should make a copy of the :file:`.spyder-py3` folder in your user home directory, and restore it afterwards if this doesn't solve the problem.
+
+#. **Try installing Spyder into a new Conda environment** (recommended) or ``virtualenv``/``venv``, and see if the issue reoccurs.
+
+In your system terminal (or Anaconda Prompt on Windows), run the following commands to create an a fresh, clean environment and start Spyder in it:
+
+.. code-block:: bash
+
+   conda create -n spyder-env spyder -y
+   conda activate spyder-env
+   spyder
+
+   If this fixes the issue, the problem was likely due to another package installed on your system, particularly if done with pip, which can cause many problems and should be avoided if at all possible.
```

Comment:
```suggestion
If this fixes the issue, the problem was likely due to another package installed on your system, particularly if done with pip, which can cause many problems and should be avoided if at all possible.
```

The indentation added to this sentence was placing it inside the code block above.

Reply:
Yup, good catch.